firmware/library/L4_Application/Temperature_Sensor.hpp

```diff
@@ -0,0 +1,21 @@
+/* Interface used to obtain the temperature of the system.*/
+#pragma once
+
+class TemperatureInterface
+
+{
+ public:
+  virtual bool Init() = 0;
+  // Initialize Drivers
+
+  virtual unsigned char
+  GetTemperatureByte() = 0;
```

Comment:
I'm not sure why this is on a newline. Also use uint8_t, rather than unsigned char. You cannot guarantee the size of unsigned char.

Reply:
For some reason lint does not like it. It kept telling me that each line should have at most 80 characters.

Reply:
If that's the case, then you need to make sure no lines exceed 80 characters. You can use new lines to do this. You can also run clang-format on your source file to reformat it.